app/pages/shows/_slug/index.vue

```diff
@@ -137,67 +153,67 @@ export default {
       const day = d.getDate().toLocaleString('en-US', { minimumIntegerDigits: 2 })
       return `${year}-${month}-${day}`
     },
-    arcsiShows () {
-      return this.$store.getters.returnArcsiShows
-    },
-    arcsiInfosBlock () {
-      if (this.arcsiShows) {
-        const allShows = [...this.arcsiShows]
-        return allShows
-          .filter(show => show.archive_lahmastore_base_url === this.$route.params.slug)
-          .shift()
+    getLatestEpisode () {
+      if (this.showObject?.items) {
+        const itemsSorted = this.showObject.items
+          .filter(show => show.play_date < this.getToday)
```

Comment:
I would use item instead of show here, a bit confusing.

Reply:
True! Let's fix these things when we come across them while coding but I won't create a new issue for such small matters.